demo/server.js

```diff
@@ -20,6 +20,7 @@ initSqlJs().then(SQL => {
     sqlstr = "CREATE TABLE model (id INTEGER PRIMARY KEY, name TEXT, address TEXT, description TEXT, model_type TEXT, encoder TEXT, accuracy NUMBER);"
       + "CREATE TABLE data (transaction_hash TEXT PRIMARY KEY, text TEXT);"
       + "CREATE INDEX index_address ON model(address);";
+      + "CREATE Table accuracy_time ( transaction_hash TEXT ,id INTEGER ,accuracy NUMBER , Timestamp DATETIME DEFAULT CURRENT_TIMESTAMP); "
```

Comment:
Thanks for setting this up!
Please keep the style consistent with the other statements. So it should be "CREATE TABLE ..." and use all lowercase column names.
As for some of these names. For the table name, just "accuracy" is fine.
I'm not sure what the purpose of the transaction_hash column is so it would be good to add a comment in the code to explain it to people when they read the code. The transaction_hash is the data table is the transcation hash in the Ethereum blockchain for when the data was added. I'm not sure if the same thing is appropriate here since the accuracy can be computed off-chain. Instead (or in addition), having a column for the block number would be good so we know as of which block the model has the recorded accuracy. 
I guess "id" is the model ID? Would you call it "model_id" and use a FOREIGN KEY: `FOREIGN KEY (model_id) REFERENCES model (id)`. See https://www.sqlitetutorial.net/sqlite-foreign-key/ for details.
"Timestamp" should be lowercase.

Sorry for sounding so picky. I'm always like this. I'm just trying to teach you some things.

Reply:
no worries, I understand. I do all the changes!